.vscode/cspell.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
Is this file needed?

Reply:
I don't know. For me, it is useful, as the IDE shows a big part of the text as spelling errors and then I miss the real mistypes. But if you think it should be removed, I can remove it.